mailo/src/main/scala/mailo/http/SMTPClient.scala

```diff
@@ -111,4 +111,21 @@ class SMTPClient(implicit conf: Config = ConfigFactory.load())
     headers.foreach(h => message.addHeader(h._1, h._2))
     internalSend(message, attachments, tags)
   }
+
+  override def sendBatch(
+    to: List[String],
+    from: String,
+    cc: Option[String],
+    bcc: Option[String],
+    subject: String,
+    content: MailRefinedContent.MailRefinedContent,
+    attachments: List[Attachment],
+    tags: List[String],
+    recipientVariables: Map[String, Map[String, String]],
+    headers: Map[String, String]
+  )(
+    implicit
+    executionContext: ExecutionContext
+  ): Future[Either[MailError, MailResponse]] =
+    throw new UnsupportedOperationException("unable to send batch messages in SMTP")
```

Comment:
I would like to eventually refactor this Into smaller interfaces, so that we don’t have to throw these exceptions